src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import React from 'react';
+import { filterLinks } from '../../helper/filterLinks';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filter: Filter,
```

Comment:
Consider using a more descriptive name for the 'onFilter' prop, like 'onFilterChange', to make it clear that this function is called when the filter changes 🤔